index.html

```diff
@@ -0,0 +1,368 @@
+              />
+            </div>
+          </div>
+          <div class="sectionfour__container__rows__row">
+            <h2>Tonic</h2>
+            <ul class="first__ul">
+              <li>FACEBOOK</li>
+              <li class="disc">Full Stack Dev</li>
+              <li class="disc">2015</li>
+            </ul>
+            <p>
+              Exploring the future of media in Facebook's first Virtual Reality
+              app; a place to discover and enjoy 360 photos and videos on Gear
+              VR.
+            </p>
+            <ul class="second__ul">
+              <li>html</li>
+              <li>Ruby</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button class="hover">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- end sectionfour -->
+    <!-- start sectionfive -->
+    <section class="sectionfive pd-y" id="sectionfive">
+      <div class="container">
+        <div class="sectionfive__container__services">
+          <div class="sectionfive__container__services__service order__2">
+            <h2>
+              Multi-Post <br />
+              Stories
+            </h2>
+            <ul class="first__ul">
+              <li>Uber</li>
+              <li class="disc">Lead Developer</li>
+              <li class="disc">2018</li>
+            </ul>
+            <p>
+              A smart assistant to make driving more safe, efficient, and fun by
+              unlocking your most expensive computer: your car.
+            </p>
+            <ul class="second__ul">
+              <li>html</li>
+              <li>Ruby</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+            <button class="hover">See Project</button>
+          </div>
+          <div class="sectionfive__container__services__service order__1">
+            <div class="sectionfive__container__services__service__responsive">
+              <img
+                src="./static/imgs/two.png"
+                alt="four"
+                class="responsive sectionfive__container__services__service__responsive__respo"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- end sectionfive -->
+    <!-- start sectionsix -->
+    <section class="sectionsix pd-y" id="sectionsix">
+      <div class="container">
+        <div class="sectionsix__container__skills">
+          <div class="sectionsix__container__skills__skill">
+            <h2>
+              About <br />
+              Myself
+            </h2>
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <button>LET’S CONNECT</button>
```

Comment:
- [ ] Kindly consider using a header tag `hx` instead of a `button` here on `connect`

-  [ ] Kindly make sure you avoid this deep nesting as they increase the length of your `HTML` and even `CSS` file for no reason